packages/foreman/foreman/foreman.spec

```diff
@@ -49,83 +49,50 @@ Requires: (%{name}-fapolicyd if fapolicyd)
+Requires: (rubygem(validates_lengths_from_database) >= 0.5 with rubygem(validates_lengths_from_database) < 1.0)
+Requires: (rubygem(friendly_id) >= 5.4.2 with rubygem(friendly_id) < 6)
+Requires: (rubygem(secure_headers) >= 6.3 with rubygem(secure_headers) < 7.0)
+Requires: (rubygem(safemode) >= 1.3.5 with rubygem(safemode) < 2)
+Requires: (rubygem(fast_gettext) >= 1.4 with rubygem(fast_gettext) < 2.0)
+Requires: (rubygem(gettext_i18n_rails) >= 1.8 with rubygem(gettext_i18n_rails) < 2.0)
+Requires: (rubygem(rails-i18n) >= 7.0 with rubygem(rails-i18n) < 8.0)
+Requires: (rubygem(logging) >= 1.8.0 with rubygem(logging) < 3.0.0)
+Requires: (rubygem(fog-core) >= 2.1 with rubygem(fog-core) < 3.0)
 Requires: rubygem(net-scp)
 Requires: rubygem(net-ssh)
 Requires: rubygem(net-ldap) >= 0.16.0
 Requires: rubygem(net-ping)
-Requires: rubygem(activerecord-session_store) >= 2.0.0
-Requires: rubygem(activerecord-session_store) < 3
-Requires: rubygem(sprockets) >= 4.0
-Requires: rubygem(sprockets) < 5.0
-Requires: rubygem(sprockets-rails) >= 3.0
-Requires: rubygem(sprockets-rails) < 4.0
-Requires: rubygem(responders) >= 3.0
-Requires: rubygem(responders) < 4.0
-Requires: rubygem(roadie-rails) >= 3.0
-Requires: rubygem(roadie-rails) < 4.0
-Requires: rubygem(deacon) >= 1.0
-Requires: rubygem(deacon) < 2.0
-Requires: rubygem(webpack-rails) >= 0.9.8
-Requires: rubygem(webpack-rails) < 0.10.0
-Requires: rubygem(mail) >= 2.7
-Requires: rubygem(mail) < 3.0
-Requires: rubygem(sshkey) >= 2.0
-Requires: rubygem(sshkey) < 3.0
-Requires: rubygem(dynflow) >= 1.6.5
-Requires: rubygem(dynflow) < 2.0.0
+Requires: (rubygem(activerecord-session_store) >= 2.0.0 with rubygem(activerecord-session_store) < 3)
+Requires: (rubygem(sprockets) >= 4.0 with rubygem(sprockets) < 5.0)
+Requires: (rubygem(sprockets-rails) >= 3.0 with rubygem(sprockets-rails) < 4.0)
+Requires: (rubygem(responders) >= 3.0 with rubygem(responders) < 4.0)
+Requires: (rubygem(roadie-rails) >= 3.0 with rubygem(roadie-rails) < 4.0)
+Requires: (rubygem(deacon) >= 1.0 with rubygem(deacon) < 2.0)
+Requires: (rubygem(webpack-rails) >= 0.9.8 with rubygem(webpack-rails) < 0.10.0)
+Requires: (rubygem(mail) >= 2.7 with rubygem(mail) < 3.0)
+Requires: (rubygem(sshkey) >= 2.0 with rubygem(sshkey) < 3.0)
+Requires: (rubygem(dynflow) >= 1.6.5 with rubygem(dynflow) < 2.0.0)
 Requires: rubygem(daemons)
-Requires: rubygem(bcrypt) >= 3.1
-Requires: rubygem(bcrypt) < 4.0
+Requires: (rubygem(bcrypt) >= 3.1 with rubygem(bcrypt) < 4.0)
 Requires: rubygem(get_process_mem)
-Requires: rubygem(rack-cors) >= 1.1
-Requires: rubygem(rack-cors) < 2.0
-Requires: rubygem(jwt) >= 2.2.2
-Requires: rubygem(jwt) < 3.0
-Requires: rubygem(graphql) >= 1.13.0
-Requires: rubygem(graphql) < 1.14.0
+Requires: (rubygem(rack-cors) >= 1.1 with rubygem(rack-cors) < 2.0)
+Requires: (rubygem(jwt) >= 2.2.2 with rubygem(jwt) < 3.0)
+Requires: (rubygem(graphql) >= 1.13.0 with rubygem(graphql) < 1.14.0)
 Requires: rubygem(graphql-batch)
+Requires: rubygem(rss)
```

Comment:
nope? :)

Reply:
Ah yes, my local gemfile for Ruby 3 strikes again.